diff --git a/arch/arm64/boot/dts/vendor/qcom/Makefile b/arch/arm64/boot/dts/vendor/qcom/Makefile
index b3450e44a..8155403c1 100644
--- a/arch/arm64/boot/dts/vendor/qcom/Makefile
+++ b/arch/arm64/boot/dts/vendor/qcom/Makefile
@@ -381,7 +381,8 @@ dtb-$(CONFIG_ARCH_KHAJE) += khaje-idp.dtb \
 		khajeg-idp-nopmi.dtb \
 		khajeg-idp-usbc.dtb \
 		khajeg-idp-pm8010.dtb \
-		khajeg-idps-display-90hz.dtb
+		khajeg-idps-display-90hz.dtb \
+		spes-khaje-idp-nopmi.dtb
 endif
 
 ifeq ($(CONFIG_BUILD_ARM64_DT_OVERLAY),y)
diff --git a/arch/arm64/boot/dts/vendor/qcom/spes-khaje-idp-nopmi-overlay.dts b/arch/arm64/boot/dts/vendor/qcom/spes-khaje-idp-nopmi.dts
similarity index 100%
rename from arch/arm64/boot/dts/vendor/qcom/spes-khaje-idp-nopmi-overlay.dts
rename to arch/arm64/boot/dts/vendor/qcom/spes-khaje-idp-nopmi.dts
